app/feature.go

```diff
@@ -100,14 +100,6 @@ func NewFeatureCommand() (CommandOut, error) {
 			Usage:   "feature commands",
 			Hidden:  true,
 			Subcommands: []*cli.Command{
-				{
-					Name:    "toggle-export",
-					Aliases: []string{"te"},
-					Usage:   "switch export on/off",
-					Action: func(c *cli.Context) error {
-						return toggleFeature(ExportFeatureFlag)
```

Comment:
`func getFeatureFlagsFromConfigFile(featureFlagConfigPath string) ([]FeatureFlag, error) { `

For the above function, if an older client has the export feature flag in their config, will the "get" command still print out export as a feature flag? If so, should we add code to protect against printing garbage values?

Reply:
Will skip printing it if we don't have this feature flag any more.